### PR TITLE
Tiles adjustment

### PR DESCRIPTION
**Pull request by lenarmathias**
Improved Tile responsiveness, added cast/crew dependency in MovieTile, implemented ".toFixed" for rating, and fixed MissingMoviePoster for mobile view (other resolutions pending).

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -19,10 +19,12 @@ export const theme = {
 
   breakpoints: {
     verySmallMobile: "375px",
+    vga: "640px",
```
Comment (Mateusz-95) [inline src/theme.js:22]: What vga mean?

File: src/common/Tiles/MovieTile/styled.js [hunk 2]
```diff
@@ -47,11 +46,20 @@ export const MoviePoster = styled.img`
...
     background-color: ${({ theme }) => theme.color.silver};
+    height: 100%;
+    width: 100%;
+    max-height: 380px;
     display: grid;
     align-items: center;
     justify-content: center;
+    border-radius: 5px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.smallMobile}) {
```
Comment (Mateusz-95) [inline src/common/Tiles/MovieTile/styled.js:59]: Do we not need to add "media" for other devices as well?

File: src/common/Tiles/PeopleTile/styled.js [hunk 3]
```diff
@@ -1,34 +1,27 @@
...
+import { boxShadowColor } from "../../styled";
 
 export const PeopleTileContainer = styled.div`
     background-color: ${({ theme }) => theme.color.white};
-    width: 208px;
-    height: 346px;
+    max-width: 208px;
+    max-height: auto;
     display: grid;
     grid-template-rows: auto 1fr 1fr;
```
Comment (Mateusz-95) [inline src/common/Tiles/PeopleTile/styled.js]: Maybe we should stick to the above patterns and write it as: auto, repeat(2, 1fr)?

**Review by Mateusz-95 — APPROVED** [review]
Good job! However, I've posted a few questions regarding 3 sections of the code.

File: src/common/Tiles/PeopleTile/styled.js (hunk 3 above)
Comment (lenarmathias) [reply src/common/Tiles/PeopleTile/styled.js]: You are right, I'm gonna change that.

File: src/theme.js (hunk 1 above)
Comment (lenarmathias) [reply src/theme.js:22]: VGA simply is 640x480 pixels resolution.

File: src/common/Tiles/MovieTile/styled.js (hunk 2 above)
Comment (lenarmathias) [reply src/common/Tiles/MovieTile/styled.js:59]: This keeps the element displaying correctly in mobile view, but it still needs further adjustments for other views.

**Pull request merged**